application/views/main/base.php

```
@@ -36,33 +36,33 @@
-			<br /><b>* PHP Version:</b>
+			<br><b>* PHP Version:</b>
 			`<?php echo htmlentities(phpversion(), ENT_QUOTES); ?>`
-			<br /><b>* DB Backend:</b>
+			<br><b>* DB Backend:</b>
 			`<?php
 				$this->load->helper('kalkun_helper');
 				$db_name_human = get_database_property($this->db->platform())['human'];
 				echo $db_name_human, ' ', $this->db->version(), ' (', $this->db->platform(), ')'; ?>`
-			<br /><b>* Gammu version:</b>
+			<br><b>* Gammu version:</b>
 			`<?php echo  filter_data(htmlentities(strval($this->Kalkun_model->get_gammu_info('gammu_version')->row('Client'))), ENT_QUOTES); ?>`
```

PHPStan (2.1.54)

[error] Function filter_data invoked with 2 parameters, 1 required.


(arguments.count)